pydocmd.yml

```diff
@@ -0,0 +1,16 @@
+site_name: 'AWS Integration Interface'
```

Comment:
This file should probably either be updated or removed (in which case, also `make_docs`).  I'm still experimenting with generating docs for layers, so could just remove it for now unless you want to generate docs for this layer similar to https://github.com/juju-solutions/interface-aws/blob/master/docs/requires.md